src/Widgets/ContactFormWidget/ContactFormWidgetComponent.js

```diff
@@ -4,99 +4,110 @@ added to contactForm.html as well. See the following link for details:
 https://www.netlify.com/blog/2017/07/20/how-to-integrate-netlifys-form-handling-in-a-react-app/
 */
 
-Scrivito.provideComponent('ContactFormWidget', ({ widget }) =>
-  <div className="row floating-label">
-    <form method="post">
-      <input
-        type="hidden"
-        name="form-name"
-        value="contact"
-      />
-      <div className="hidden">
-        <label>Don’t fill this out: <input name="bot-field" /></label>
-      </div>
-      <div className="col-sm-6">
-        <div className="form-group">
-          <label htmlFor="contactName">Your name</label>
-          <input
-            className="form-control input-lg"
-            id="contactName"
-            name="contactName"
-            placeholder="Name"
-            type="text"
-            required
-          />
+Scrivito.provideComponent('ContactFormWidget', ({ widget }) => {
+  const classNames = ['row'];
+  if (widget.get('isTransparent') === 'yes') {
+    classNames.push('panel-white-transparent');
+  } else {
+    classNames.push('floating-label');
```

Comment:
You learn sth new every day: https://stackoverflow.com/a/14723896
See also the ES6 answer